### DISCUSSION TITLE
Releases API - Get all releases for a repository

### DISCUSSION THREAD

**Discussion by robertmaynard** [community/community#24887] [Apps, API and Webhooks]
<p>I am currently using <code>repos/:owner/:repo/releases</code> to find all the releases for a given project, but it currently only returns the last 30 releases for a project. I looked at using <code>repos/:owner/:repo/tags</code> instead but it also limits me to the last 30 releases.</p>
<p>What is the best API to use to collect all tags / releases a project has? </p>

**Comment by robertmaynard** [comment, marked as answer]
<p>I figured out my problem. I wasn’t supportingpagination ( <a href="https://developer.github.com/v3/#pagination" rel="nofollow noopener">https://developer.github.com/v3/#pagination</a> ).</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/robertmaynard">@robertmaynard</a>,</p>
<p>Thank you for sharing what worked! We’re looking forward to seeing you around!</p>
<p>Cheers,</p>
<p>Andrea</p>
